dev-requirements.txt

```diff
@@ -0,0 +1,7 @@
+-r requirements.txt
+ninja
+flake8
+numpy
+scipy
+mkl
```

Comment:
if you're in a conda env, ideally these come in from anaconda.

Reply:
How? When I setup a new conda env (using `https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh`) and clone a new pytorch I always have to manually pip install them, and then I always forgot some so it all takes time